VPC/output.tf

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Can you add comment line where it will show what value you will get from these output block